### PR TITLE
RHDEVDOCS-3119 Remove old warnings from modules/installation-initiali…

### PR DESCRIPTION
**Pull request by rolfedh**
…zing.adoc in master

- Aligned team: Dev Tools
- For branches: master ONLY
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3119
- Direct link to doc preview:  https://deploy-preview-33862--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-customizations.html#installation-initializing_installing-rhv-customizations
- SME review: @Gal-Zaidman 
- QE review: Removing obsolete content from master, not the current docs. QE review not required. 
- Peer review: @bergerhoffer @stoobie
- <All reviews complete. Please merge now>

### CODE REVIEW

File: modules/installation-initializing.adoc [hunk 1]
```diff
@@ -150,20 +150,6 @@ endif::osp[]
 ifdef::vsphere,vmc[]
 VMware vSphere.
 endif::vsphere,vmc[]
-ifdef::rhv[]
```
Comment (bergerhoffer) [inline modules/installation-initializing.adoc:153]: I don't think you want to remove this ifdef/endif that contains the RHV name.

Comment (stoobie) [reply modules/installation-initializing.adoc:153]: +1. The RHV name here is part of the sentence that precedes the ifdef. It should stay. Pending SME approval, the two warnings can go from master, but they need to stay in 4.6 (and earlier)

**Review by stoobie — CHANGES_REQUESTED** [review]
Pending SME approval and one comment, LGTM.

**Comment by Gal-Zaidman** [comment]
I'm looking at the original docs and I don't see those warnings, why is that?
https://docs.openshift.com/container-platform/4.7/installing/installing_rhv/installing-rhv-customizations.html#installation-initializing_installing-rhv-customizations

**Comment by stoobie** [comment]
> I'm looking at the original docs and I don't see those warnings, why is that?
> https://docs.openshift.com/container-platform/4.7/installing/installing_rhv/installing-rhv-customizations.html#installation-initializing_installing-rhv-customizations

I only see it in 4.5:
https://docs.openshift.com/container-platform/4.5/installing/installing_rhv/installing-rhv-customizations.html#installation-initializing_installing-rhv-customizations

**Comment by rolfedh** [comment]
> I'm looking at the original docs and I don't see those warnings, why is that?

@Gal-Zaidman @stoobie The warnings were a temporary stop-gap to deal with a bug that was fixed shortly after it appeared. They weren't very relevant to 4.6 and 4.7 users. I don't recall the details, but I did some fancy footwork with making changes in various PRs and obviously ended up tripping over my own toes. This PR is me cleaning up that small mess.

File: modules/installation-initializing.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/installation-initializing.adoc:153]: Thanks. Updated.

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

Because this is just removing warnings from master that don't existing in 4.8, 4.7, etc., I don't think that this requires QE.

**Pull request merged**